taskcluster/ci/config.yml

```diff
@@ -142,9 +142,11 @@ scriptworker:
     scope-prefix: project:xpi:releng
     signing-format:
         by-xpi-type:
-            privileged: privileged_webextension
-            system: system_addon
+            hosted-privileged: privileged_webextension
```

Comment:
I don't love the name `hosted` -- from my perspective it's not clear where it's hosted. Is Normandy hosted? Is S3 hosted, if it's served off Balrog or Normandy? What is the benefit of this name compared to amo?

Reply:
That's a good point, maybe we can come up with a better name. We have run into confusion in the past where things were self-hosted (on a website off AMO), so the assumption was that the AMO process and policies wouldn't apply. The intent here is to cover that, so that it's clear that any (actual add-on, to be manually installed by a user), falls under that, regardless of whether it's distributed from AMO or from an own landing page.

Reply:
Hm.

We could:

- find a new name
- document what either `amo-privileged` or `hosted-privileged` means somewhere, and point at that
- leave `{amo,hosted}-privileged` as `privileged` -- anything that's not mozillaonline or normandy follows the default `privileged` rules.

Reply:
I'd prefer the first or second option (note that I already added some low-level documentation in https://github.com/mozilla-extensions/xpi-template/pull/18/files#diff-ac0229d1171c0983b27003af4c7441a5R10-R13).
The third option might be confusing because those two groups should be distinct, where as one could assume `privileged` is a superset of `{normandy|<anything>}-privileged`